travis_deploy.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -e
+set -x
+
+TAG=$TRAVIS_TAG
+BRANCH=$TRAVIS_BRANCH
+PR=$TRAVIS_PULL_REQUEST
+
+echo $TAG
+echo $BRANCH
+echo $PR
+
+if [ -z $TAG ]
+then
+    echo "No tags, tagging as: latest"
+    TAG="latest"
+fi
+
+# if this is on the master branch and this is not a PR, deploy it
+if [ $BRANCH = "master" # -a $PR = "false" ]
```

Comment:
@nkhumphreys I think this should be develop in our case, unless we are adding this to the PR we are sending to rapidpro repo, then we can decide otherwise.